datalad/tests/utils.py

```diff
@@ -273,8 +273,8 @@ def  _wrap_skip_if(*args, **kwargs):
 
 
 def skip_ssh(func):
-    """Skips SSH tests if on windows or if environment variable
-    DATALAD_TESTS_SSH was not set
+    """Skips SSH tests if datalad.tests.ssh (env var DATALAD_TESTS_SSH)
+    is not set at all or set to '0', 'false', or 'no'.
```

Comment:
this and below look unrelated to this PR.  I guess (didn't look) the base might have been `maint` but PR made against `master`?

Reply:
You are right. The branch was created on maint and I forgot to change the PR to be against maint